README.md

```diff
@@ -55,8 +55,9 @@ After you have a **MicrosoftGraphServiceClient** that is authenticated, you can
 To retrieve the user's drive:
 
 ```ruby
-result = client.me.drive.get.resume
-puts "Found Drive : " + result.id
+user_item_req_builder = client.users_by_id '9327b2bb-74b9-4d22-9839-eff4a982f541'
+result = user_item_req_builder.get.resume
```

Comment:
```suggestion
result = client.users_by_id('<user id>').drive.get.resume
```

what do you think of this instead to stay in the lines of the description for the example?

Reply:
wow, blazingly fast review, thanks so much. Yes your suggestion sounds great.